row.py

```diff
@@ -795,6 +796,143 @@ def download_run(bucket, run_name):
     return location.joinpath(run_name)
 
 
+def download_ocr_results(bucket_name, run_name, out_dir):
+    """download ocr results from a GCP bucket
+
+    Args:
+        bucket (str): the name of the bucket
+        run_name (str): the name of the run to get files from
+        out_dir (str): where to save the results
+
+    Returns:
+        str: the location of the files
+    """
+    if bucket_name.startswith("gs://"):
+        bucket_name = bucket_name[5:]
+
+    if "STORAGE_CLIENT" not in globals():
+        STORAGE_CLIENT = google.cloud.storage.Client()
+    bucket = STORAGE_CLIENT.bucket(bucket_name)
+    blobs = bucket.list_blobs(prefix=run_name)
+    location = Path(__file__).parent / "data"
+
+    if not location.joinpath(f"ocr_results/{run_name}").exists():
+        location.joinpath(f"ocr_results/{run_name}").mkdir(parents=True)
+
+    ocr_dir = location.joinpath(f"ocr_results/{run_name}")
```

Comment:
```suggestion
    ocr_dir = location.joinpath(f"ocr_results/{run_name}")
    ocr_dir.mkdir(parents=True, exist_ok=True)
```

**suggestion**: it's silly to create the same path twice before putting it in a variable. The if check is unnecessary since mkdir can run with exists_ok whether the directories exist or not.

Reply:
done